.babelrc

```diff
@@ -2,9 +2,9 @@
 	"presets": [
 		"@babel/preset-env"
 	],
-	"targets": {
-		"node": "6.9.0"
-	},
+	// "targets": {
```

Comment:
@szandany i did this. need to undo this before shipping and make it work on node v14 (which is what you have installed on your machine)